airbyte-integrations/connectors/source-zendesk-support/unit_tests/unit_test.py

```diff
@@ -1120,7 +1120,7 @@ def test_read_ticket_metrics_with_error(self, requests_mock):
                 (404, ResponseAction.IGNORE),
                 (403, ResponseAction.IGNORE),
                 (500, ResponseAction.RETRY),
-                (429, ResponseAction.RETRY),
+                (429, ResponseAction.RATE_LIMITED),
```

Comment:
should this be removed from the final merge since we need to first cut a release and bump the version?

Reply:
agree 
updated

Reply:
updated